README.md

```diff
@@ -1,9 +1,11 @@
-# Marsha
+# Marsha AI Language
 
 [![Discord Follow](https://dcbadge.vercel.app/api/server/p5BTaWAdjm?style=flat)](https://discord.gg/p5BTaWAdjm)
 [![GitHub Repo Stars](https://img.shields.io/github/stars/alantech/marsha?style=social)](https://github.com/alantech/marsha)
 
-Marsha is a functional, higher-level, English-based programming language that gets compiled into tested Python software by an LLM. This repository contains an LLM-based compiler that implicitly defines the Marsha syntax.
+<p align="center"><b>Describe Logic ⴲ Provide Examples ⴲ Run Reliably</b><br /><br /><img src="./examples/images/duckduckgo-terminal.gif" /></p>
```

Comment:
oh shouldn't we keep here the markdown definition?

Reply:
I wanted it centered with the text, so that's why I kept it inside of the `<p>` block

Reply:
oh no, I mean the `duckduckgo.mrsh` content

Reply:
Ahh!

Can we prefix the `marsha duckduckgo.mrsh` with a `cat duckduckgo.mrsh` and just show it in the terminal GIF?